app/components/cookies_banner_component.rb

```diff
@@ -1,13 +1,13 @@
 class CookiesBannerComponent < ApplicationComponent
-  attr_reader :create_path, :reject_path, :preferences_path
+  attr_reader :accept_path, :reject_path, :preferences_path
```

Comment:
This is a non-functional change. I decided to replace `create` with `accept` to reflect better the user action of accepting cookies.